src/cmd/helm.rs

```diff
@@ -524,6 +526,105 @@ impl Helm {
 
         Ok(())
     }
+
+    pub fn prepare_chart_backup<P>(
```

Comment:
Move this outside of the helm module/crate.
It is not part of the functionality of the helm command.

Just create a helper function that does this for you, outside of this module.